README.md

```diff
@@ -100,76 +140,122 @@ spec:
   type: appregistry
   endpoint: https://quay.io/cnr
   registryNamespace: pmacik
+EOS
 ```
 
-  * For example, Creation of a PostgreSQL operator-managed database exposes the following binding
-    information:
+Then navigate to the `Operators`->`OperatorHub` in the OpenShift console and in the `Database` category select the `PostgreSQL Database` operator
 
-``` yaml
+![PostgreSQL Database Operator as shown in OperatorHub](assets/operator-hub-pgo-screenshot.png)
+
+and install a `stable` version.
+
+This makes the `Database` custom resource available, that the application developer will use later.
+
+### Application Developer
+
+#### Create a namespace called `service-binding-demo`
+
+The application and the DB needs a namespace to live in so let's create one for them:
+
+``` bash
+cat <<EOS |kubectl apply -f -
+---
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: service-binding-demo
+EOS
+```
+
+#### Import an application
+
+In this example we will import an arbitrary [Node.js application](https://github.com/pmacik/nodejs-rest-http-crud).
+
+In the OpenShift Console switch to the Developer perspective. (Make sure you have selected the `service-binding-demo` project). Navigate to the `+ADD` page from the menu and then click on the `[Import from Git]` button. Fill in the form with the following:
+
+* `Git Repo URL` = `https://github.com/pmacik/nodejs-rest-http-crud`
+* `Project` = `service-binding-demo`
+* `Application`->`Create New Application` = `NodeJSApp`
+* `Name` = `nodejs-app`
+* `Builder Image` = `Node.js`
+* `Create a route to the application` = checked
+
+and click on the `[Create]` button.
+
+Notice, that during the import no DB config was mentioned or requestd.
+
+When the application is running navigate to it's route to verify that it is up. Notice that in the header it says `(DB: N/A)`. That meens that the application is not connected to a DB and so it should not work properly. Try the application's UI to add a fruit - it causes an error proving that the DB is not connected.
```

Comment:
Small grammar nit:  Should be "its" not "it's"

Reply:
Thanks @ldimaggi, Fixed